uktrade/great-cms
# EAM-47: Logged in listings page

Logged in listing page - https://uktrade.atlassian.net/browse/EAM-47

**To test**
- Create a registration instance in your local django shell (`make manage shell_plus`)
- Add `FEATURE_EXPORT_ACADEMY_RELEASE_2=true` to your secrets_do_not_commit file
- Make sure you've got directory-forms-api running locally
- Go to http://greatcms.trade.great:8020/export-academy/events/

**Screenshots**

**Showing one booked and one not booked event**
![Screenshot 2023-05-02 at 11 25 34](https://user-images.githubusercontent.com/22460823/235642647-38dfcb6c-edda-4708-9547-e75a8a0b3132.png)

**Showing date filters greyed out when 'Current' or 'Past' event filters selected**
![Screenshot 2023-05-02 at 11 25 50](https://user-images.githubusercontent.com/22460823/235642756-aafe1a13-5874-4242-87cb-d0ce6ad7d9a2.png)

### Workflow


### Merging
